cloud-azure/src/main/resources/templates/arm-dbstack.ftl

```diff
@@ -0,0 +1,179 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "dbServerName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the database server resource."
+            }
+        },
+        "skuTier": {
+            "type": "string"
+        },
+        "skuFamily": {
+            "type": "string"
+        },
+        "skuCapacity": {
+            "type": "int"
+        },
+        "skuSizeMB": {
+            "type": "int"
+        },
+        "skuName": {
+            "type": "string",
+            "defaultValue": "GP_Gen5_2",
+            "allowedValues": [
+                "GP_Gen5_2",
+                "GP_Gen5_4",
+                "GP_Gen5_8",
+                "GP_Gen5_16",
+                "GP_Gen5_32",
+                "MO_Gen5_2",
+                "MO_Gen5_4",
+                "MO_Gen5_8",
+                "MO_Gen5_16",
+                "MO_Gen5_32"
+            ],
+            "metadata": {
+                "description": "Azure database for PostgreSQL sku name "
+            }
+        },
+        "dbVersion": {
+            "type": "string",
+            "metadata": {
+                "description": "Server version."
+            }
+        },
+        "adminLoginName": {
+            "type": "securestring",
+            "minLength": 1,
+            "metadata": {
+                "description": "The administrator login name for the database server."
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "minLength": 8,
+            "maxLength": 128,
+            "metadata": {
+                "description": "The administrator password for the database server."
+            }
+        },
+        "sslEnforcement": {
+            "type": "string",
+            "defaultValue": "Enabled",
```

Comment:
What do you think about making this a Boolean value? Is it possible and straightforward to have the template engine convert true/false to Enabled/Disabled? I worry that the spelling will be wrong at some point. :)

Same for geoRedundantLookup, storageAutogrow.

Reply:
Yes, would be nice, I check it now. On the other hand, field "allowedValues" does a sanity check on the input string right away. The feedback comes before anything in azure would be provisioned.

Reply:
Ok, it was quite easy to do the boolean mapping, so fixed.